src/miniscript/astelem.rs

```diff
@@ -356,8 +367,14 @@ where
                 if wrap.is_empty() {
                     return Err(Error::Unexpected(top.name.to_owned()));
                 }
-                frag_name = name;
-                frag_wrap = wrap;
+                if name == "pk" {
+                    frag_name = "pk_k";
+                    aliased_wrap = wrap.to_owned() + &"c";
```

Comment:
As a matter of style, I don't like using `+` on strings, because I think it is an abuse of operator overloading to use `+` for a noncommutative operator.